src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/client/ModuleBaritone.kt

```diff
@@ -0,0 +1,352 @@
+                    createSetting("Jump", jumpPenalty)
+                }
+
+                treeAll(
+                    Blocks
+                )
+            }
+        }
+
+        private object Blocks : Configurable("Blocks") {
+            init {
+                if (BaritoneUtil.isAvailable) {
+                    with(BaritoneAPI.getSettings()) {
+                        createSetting("Placement", blockPlacementPenalty, maxRangedValue = 300)
+                        createSetting("BreakAdditional", blockBreakAdditionalPenalty)
+                        createSetting("BreakCorrectBlockMultiplier",
+                            breakCorrectBlockPenaltyMultiplier, maxRangedValue = 20)
+                        createSetting("PlaceIncorrectBlockMultiplier", placeIncorrectBlockPenaltyMultiplier)
+                    }
+                }
+            }
+        }
+    }
+
+    private object Elytra : Configurable("Elytra") {
+        init {
+            if (BaritoneUtil.isAvailable) {
+                with(BaritoneAPI.getSettings()) {
+                    createSetting("SimulationTicks", elytraSimulationTicks, 1, 100)
+                    createSetting("PitchRange", elytraPitchRange, 1, 100)
+                    createSetting("MinimumAvoidance", elytraMinimumAvoidance, -10)
+                    createSetting("PredictTerrain", elytraPredictTerrain)
+                    createSetting("EmergencyLand", elytraAllowEmergencyLand)
+                    createSetting("LandOnNetherFortress", elytraAllowLandOnNetherFortress)
+                }
+
+                treeAll(
+                    Auto,
+                    Firework,
+                    Render
+                )
+            }
+        }
+
+        private object Auto : Configurable("Auto") {
+            init {
+                if (BaritoneUtil.isAvailable) {
+                    with(BaritoneAPI.getSettings()) {
+                        createSetting("Jump", elytraAutoJump)
+                        createSetting("Swap", elytraAutoSwap)
+                    }
+                }
+            }
+        }
+
+        private object Firework : Configurable("Firework") {
+            init {
+                if (BaritoneUtil.isAvailable) {
+                    with(BaritoneAPI.getSettings()) {
+                        createSetting("Speed", elytraFireworkSpeed)
+                        createSetting("SetbackUseDelay", elytraFireworkSetbackUseDelay)
+                        createSetting("Conserve", elytraConserveFireworks)
+                    }
+                }
+            }
+        }
+
+        private object Render : Configurable("Render") {
+            init {
+                if (BaritoneUtil.isAvailable) {
+                    with(BaritoneAPI.getSettings()) {
+                        createSetting("Raytraces", elytraRenderRaytraces)
+                        createSetting("HitboxRaytraces", elytraRenderHitboxRaytraces)
```

Comment:
if this is dependent on the previous, it should be a setting for that instead instead (.../Render/HotboxRaytraces -> .../Render/Raytraces/Hitbox)

Reply:
Ill move it if I add support for ToggleableConfigurable.